### ISSUE TITLE
Method depricated problem: torch.nn.utils.weight_norm,SOS

### ISSUE THREAD

**Issue by ffrgtq** [huggingface/parler-tts#133]
Guys I'm a beginner who want to run this model in my PC(win11), but I soon faced a depricated problem as follow: C:\Anaconda\Lib\site-packages\torch\nn\utils\weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.   WeightNorm.apply(module, name, dim)
Maybe I should change certain "import" sentences, but the reference to the method is encapsulated under other package methods, I don't know if I should simply change the import sentence in specific package method and 'reconda' the related dependency......And I can only find a sentence like this " from torch import _weight_norm", and there is an extra underline than the error method name lol... 
Did you guys ever meet problem like that? If you have any solution, can please teach me? thank you a lot!

**Comment by newpolygons** [comment]
This is a warning not an error so the code should still run after this message appears. Seeing as though you are a beginner I wouldn't bother with it too much. Once this repository is updated they will likely switch over to the new function.

**Comment by ffrgtq** [comment]
OK, I'll try it again, thank you for explaining!
